### DISCUSSION TITLE
How do I customize Metrics in oap 9.2.0

### DISCUSSION THREAD

**Discussion by zhang-zhipeng** [apache/skywalking#9899] [Q&A] [answered]
I know how to add meters on the skywalking ui and how to use APIs to add meters in applications. But i don't know how to configure the receive indicator in oap. I saw similar problems in the early discussion version, but the document is no longer accessible. I don't know how to configure the instrument in version 9.2.0

**Comment by wu-sheng** [comment, marked as answer]
Have you ever read the 9.2.0 document? This question doesn't make sense.

**Reply by zhang-zhipeng** [reply]
Thank you. I have found it in the document. But I came across a new problem. I need to count the average response time of all services under the team, such as from (Service. latency) LongAvg() only queries the average response time of the current service. How can I query the average response time of services with the same prefix

**Reply by zhang-zhipeng** [reply]
I think I may be able to add a team scope or filter the service so that I can implement this functionI think I may be able to add a team scope or filter the service so that I can implement this function.

**Reply by wu-sheng** [reply]
That is a tenancy scope thing, it is out of the community's interests, I am afraid. You need to take care of it on your own.

**Reply by zhang-zhipeng** [reply]
oh,thank you
